contracts/evoting/controller/action.go

```diff
@@ -632,6 +558,86 @@ func (a *scenarioTestAction) Execute(ctx node.Context) error {
 	return nil
 }
 
+func setupSimpleElection(ctx node.Context,secret kyber.Scalar,proxyAddr1 string,serdecontext serde.Context, electionFac types.ElectionFactory,service ordering.Service) (error,string,types.Election,[]byte) {
+	fmt.Fprintln(ctx.Out, "Create election")
+
+	// Define the configuration
+	configuration := fake.BasicConfiguration
+
+	createSimpleElectionRequest := ptypes.CreateElectionRequest{
+		Configuration: configuration,
+		AdminID:       "adminId",
+	}
+
+	signed, err := createSignedRequest(secret, createSimpleElectionRequest)
+	if err != nil {
+		return createSignedErr(err),"",types.Election{},nil 
+	}
+
+	fmt.Fprintln(ctx.Out, "create election js:", signed)
+
+	url := proxyAddr1 + electionPath
+	fmt.Fprintln(ctx.Out, "POST", url)
+
+	resp, err := http.Post(url, contentType, bytes.NewBuffer(signed))
+	if err != nil {
+		return xerrors.Errorf(FAIL_RETRIEVE_DECRYPTION_STR, err),"",types.Election{},nil 
+	}
+
+	if resp.StatusCode != http.StatusOK {
+		buf, _ := io.ReadAll(resp.Body)
+		return xerrors.Errorf(UNEXPECTED_STATUS_STR, resp.Status, buf),"",types.Election{} ,nil 
+	}
+
+	body, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return xerrors.Errorf("failed to read the body of the response: %v", err),"",types.Election{} ,nil 
+	}
+
+	fmt.Fprintln(ctx.Out, "response body:", string(body))
+
+	resp.Body.Close()
+
+	var electionResponse ptypes.CreateElectionResponse
+
+	err = json.Unmarshal(body, &electionResponse)
+	if err != nil {
+		return xerrors.Errorf("failed to unmarshal create election response: %v - %s", err, body),"",types.Election{} ,nil 
+	}
+
+	electionID := electionResponse.ElectionID
+
+	electionIDBuf, err := hex.DecodeString(electionID)
+	if err != nil {
+		return xerrors.Errorf("failed to decode electionID '%s': %v", electionID, err),"",types.Election{} ,nil 
+	}
+
+	election, err := getElection(serdecontext, electionFac, electionID, service)
+	if err != nil {
+		return xerrors.Errorf(getElectionErr, err),"",types.Election{} ,nil 
+	}
+
+	// sanity check, the electionID returned and the one stored in the election
+	// type must be the same.
+	if election.ElectionID != electionID {
+		return xerrors.Errorf("electionID mismatch: %s != %s", election.ElectionID, electionID),"",types.Election{} ,nil
+	}
+
+	fmt.Fprintf(ctx.Out, "Title of the election: "+election.Configuration.MainTitle)
+	fmt.Fprintf(ctx.Out, "ID of the election: "+election.ElectionID)
+	fmt.Fprintf(ctx.Out, "Status of the election: "+strconv.Itoa(int(election.Status)))
+
+	return nil,electionID,election,electionIDBuf
+}
+
+
+func delaDebug(title,ID,status string){
```

Comment:
The function name might not be the best choice since you log at an "Info" level. A best name might be `logElectionStatus` or `logElectionInfos`, or do you have other ideas?